webroot/js/responsive/app.module.js

```diff
@@ -68,5 +68,15 @@
                     }
                 });
             };
+        })
+        .directive('resetButton', function(){
+            return function(scope, element, attrs) {
+                element.bind('click', function(e) {
+                    var target = this.getAttribute('target');
```

Comment:
You can use `attrs.target` instead of `this.getAttribute('target')`.
The `attrs` parameter will contain all the attributes of the element.